build_tools/github_actions/fetch_test_configurations.py

```diff
@@ -0,0 +1,94 @@
+"""
+This script determines what test configurations to run
+
+Required environment variables:
+  - THEROCK_BIN_DIR
+  - PLATFORM
+"""
+
+import json
+import os
+from configure_ci import set_github_output
+
+THEROCK_BIN_DIR = os.getenv("THEROCK_BIN_DIR")
+
+test_matrix = {
+    # BLAS tests
+    "rocblas": {
+        "name": "rocblas",
+        "artifact_flags": "--blas --tests",
+        "timeout": 5,
+        "executable_command": f"{THEROCK_BIN_DIR}/rocblas-test --yaml {THEROCK_BIN_DIR}/rocblas_smoke.yaml",
+        "platform": ["linux", "windows"],
+    },
```

Comment:
These field names could also use more context. Optimize for future readers of the code, not writers who want to save a few characters.

```suggestion
    "rocblas": {
        "job_name": "rocblas",
        "fetch_artifact_args": "--blas --tests",
        "timeout_minutes": 5,
        "test_command": f"{THEROCK_BIN_DIR}/rocblas-test --yaml {THEROCK_BIN_DIR}/rocblas_smoke.yaml",
        "platform": ["linux", "windows"],
    },
```